ports/stm32/boards/Passport/modules/flows/fcc_test_flow.py

```diff
@@ -16,9 +16,8 @@
 from uasyncio import sleep_ms
 from errors import Error
 
-CAMERA_DISPLAY_DURATION_SECS = 5 * 1000
-FILE_COPY_REPEAT_COUNT = 10
-FILE_SIZE_TO_CREATE = 250 * 1024
+_CAMERA_DISPLAY_DURATION_SECS = const(5 * 1000)
```

Comment:
Not a fan of the leading underscore for "internal" constants.  I know Python people like it, but it's ugly, harder to read, and more work to type.

Reply:
Yeah, it's ugly I agree but sadly that's the way to save some bytes per global constant